volmdlr/faces.py

```diff
@@ -3571,34 +3567,39 @@ def merge_with(self, other_bspline_surface3d):
 
                 bsplines_new[i] = surfaces[errors.index(min(errors))]
 
-            xmin, xmax, ymin, ymax = [0] * len(bsplines_new), [1] * len(bsplines_new), [0] * \
-                len(bsplines_new), [1] * len(bsplines_new)
+            # xmin, xmax, ymin, ymax = [0] * len(bsplines_new), [1] * len(bsplines_new), [0] * \
+            #     len(bsplines_new), [1] * len(bsplines_new)
```

Comment:
I would recommend deleting commented code. If you ever want to come back to what it was before, git could help you with it.

Reply:
Comments have been deleted !